file_handler.rb

```diff
@@ -0,0 +1,25 @@
+require 'json'
+class FileHandler
+  attr_accessor :path, :store
+
+  def initialize(path)
+    @path = path
+  end
+
+  def write_to_file(data)
+    file = File.open(@path, 'w')
+    # @store << data
+    file.write(data)
+    file.close
+  end
+
```

Comment:
- [ ] Great work implementing this feature. Currently, the data is saved to the _**JSON**_ file during the running of the application. The requirements state that data should be saved to the _**JSON**_ file on exit from your application. Kindly configure your code in such a way that the _**methods**_ that save the data to the _**JSON**_ files will be called on exit of the application. Please implement this for the methods that save people, books and rentals to the _**JSON**_ files. 😎 💻 

![image](https://github.com/blueberry1312/school-library/assets/77106962/a4c64bd3-4456-46da-8dd3-9cfbfebf4a81)